icu4j/main/classes/core/src/com/ibm/icu/number/NumberFormatterSettings.java

```diff
@@ -486,6 +492,50 @@ public T scale(Scale scale) {
         return create(KEY_SCALE, scale);
     }
 
+   /**
```

Comment:
I think this is missing paragraph breaks. `ant docs` seems broken for me so I can't easily check. :-( Just editing this to HTML to follow the other examples above.

Reply:
Fixed in 5f1379db19a639fb5cb00527f8b3eea49955e50f if I didn't miss anything. (E.g. I've not checked generated docs.)